resources/views/store/edit_registration.blade.php

```diff
@@ -196,10 +196,13 @@
                         </div>
                     @endif
                 </div>
-                <div class="print-button">
-                    <button onclick="window.open( '{{ URL::route("store.registration.print", ["id" => $registration->id]) }}'); return false">
+                <div>
+                    <button class="long-button" onclick="window.open( '{{ URL::route("store.registration.print", ["id" => $registration->id]) }}'); return false">
                         Print a 4 week collection sheet for this family
                     </button>
+                    <button class="short-button" onclick="javascript:window.location.href='{{ URL::route("store.registration.voucher-manager", ['id' => $registration->id ]) }}'; return false;">
+                      Go to voucher manager
```

Comment:
Just "Vouchers" as per search page?

Reply:
@charlesstrange2 Nah, it's a long button above to a long button, so best to go verbose on this page